opencga-core/src/main/java/org/opencb/opencga/core/common/VersionUtils.java

```diff
@@ -0,0 +1,125 @@
+package org.opencb.opencga.core.common;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class VersionUtils {
+
+    public static List<String> order(List<String> versions) {
+        return versions.stream().map(Version::new).sorted().map(Version::toString).collect(Collectors.toList());
+    }
+
+    public static boolean isMinVersion(String minVersion, String version) {
+        int c = new Version(minVersion).compareTo(new Version(version));
+//        if (c == 0) {
+//            System.out.println(minVersion + "\t=\t" + version);
+//        } else if (c < 0) {
+//            System.out.println(minVersion + "\t<\t" + version);
+//        } else {
+//            System.out.println(minVersion + "\t>\t" + version);
+//        }
+        return c <= 0;
+    }
+
+//    public static void main(String[] args) {
```

Comment:
This code should be removed and added as part of a junit test, right?

Reply:
totally right